src/services/broadcastdiscoveryservice.cc

```diff
@@ -262,6 +262,9 @@ BroadcastDiscoveryService::createResult(
 	 * any case should not be treated here */
 	auto bdp = BroadcastDiscoveryPack::fromSerializedString(uData, ec);
 
+    if(!bdp)
```

Comment:
This seems ended up here spuriously, this fix is already included in master AFAIK